backend/uclapi/workspaces/occupeye/cache.py

```diff
@@ -122,7 +122,7 @@ def cache_survey_data(self):
                 location_data = self._const.SURVEY_LOCATIONS[survey["Name"]]
             else:
                 location_data = {
-                 "lat": "", "long": "", "address": ["", "", "", ""]
+                    "lat": "", "long": "", "address": ["", "", "", ""]

```

Comment:
unrelated change?

Reply:
This would be the hound auto-commit function formatting the entire file, this is visual indentation PEP8 compliance.